server/paymentProviders/stripe/webhook.ts

```diff
@@ -147,7 +147,6 @@ export const paymentIntentProcessing = async (event: Stripe.Event) => {
   await sequelize.transaction(async transaction => {
     const order = await models.Order.findOne({
       where: {
-        status: [OrderStatuses.NEW, OrderStatuses.PROCESSING],
```

Comment:
What other status are we expecting here? I think we should keep the list of expected status.

Reply:
It could be ERROR or ACTIVE if its a processing recurring order.